cfme/common/vm.py

```diff
@@ -185,7 +186,15 @@ def delete(self, cancel=False, from_details=False):
     @property
     def ip_address(self):
         """Fetches IP Address of VM"""
-        return self.mgmt.ip
+        mgmt_ip = None
+        pingable = [ip for ip in self.mgmt.all_ips if is_pingable(ip)]
```

Comment:
I found that every ping won't last more than 2 sec but I'd anyway took first reachable instead of iterating thru all possible ips.

Reply:
The time cost of this change, in general, was certainly on my mind - and I almost updated is_pingable to reduce TTL, but wanted to retain some flexibility in "waiting" for a response on less than ideal networks.

Very good point about not using comprehension here in order to return first.

Reply:
Part of the intent here, but I don't think its worth the time penalty, was to record _all_ the reachable IP's, in case of some VM with multiple reachable, but the 'wrong' one picked.

Reply:
Good call overall, I moved this to `cfme.utils.net.find_pingable` and have it returning first reachable.